source/BatchDrawList.h

```diff
@@ -29,6 +29,12 @@ class Sprite;
 // This class collects a set of OpenGL draw commands to issue and groups them by
 // sprite, so all instances of each sprite can be drawn with a single command.
 class BatchDrawList {
+private:
+	class DrawElement {
+	public:
+		std::vector<float> vertices;
```

Comment:
Comments here, including the one above `data` that now apply here.

Reply:
done